docker-compose.yml

```
@@ -8,30 +8,25 @@ secrets:
     file: ./src/secrets/quote.txt
 
 services:
-  example:
+  pca-report-library:
     # Run the container normally
     build:
       # VERSION must be specified on the command line:
       # e.g., --build-arg VERSION=0.0.1
       context: .
       dockerfile: Dockerfile
-    image: cisagov/example
+    image: cisagov/pca-report-library
```

We encourage using version tags on all of our Docker images whenever possible:
```suggestion
    image: cisagov/pca-report-library:0.0.1
```

---

Adding onto that, if this is just Dockerizing [cisagov/pca-report-library](https://github.com/cisagov/pca-report-library) then the image version should mirror the package version.

---

See https://github.com/cisagov/pca-report-generator-docker/pull/10#discussion_r746756361 for why this was a bad idea and a version tag should not be present on the images here.